apps/style/common.scss

```diff
@@ -825,6 +825,49 @@ input[type="radio"] {
   font-size: large;
 }
 
+#puzzleRatingButtons {
+  hr {
+    border-top: 5px dotted #ccc;
+  }
+
+  p {
+    float: left;
+    font-size: 18px;
+    color: #9E9D9D;
+    margin-right: 10px;
+  }
+
+  svg {
+    pointer-events: all;
+    fill: #9E9D9D;
+    padding: 0 5px;
+  }
+
+  #like {
+    &:hover {
+      fill: #333333;
```

Comment:
Maybe intro variable names for the colors? Could add just in the `#puzzleRatingButtons` scope if they're likely used only for this, otherwise could throw in colors.scss